recipes/coiled/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "coiled" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 498f02c2e2bda0122088f60f1826efcf15f0a21c7a604411bf7e7fee2538c3ce
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - aiohttp
+    - aiobotocore >=0.12.0,<0.13
+    - distributed >=2.19.0
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - coiled
+
+about:
+  home: https://coiled.io
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Deploy Dask cluster using Coiled Cloud'
+
+  description: |
+    A Python package to easily create, manage, and connect to Dask deployments
+    using Coiled Cloud.
+  doc_url: https://coiled.io/cloud
+
+extra:
+  recipe-maintainers:
+    - jrbourbeau
+    - mrocklin
```

Comment:
@mrocklin - are you willing to be a maintainer? I assume so, but we need to have it explicitly OK'd